packages/vitest/src/node/plugins/envReplacer.ts

```diff
@@ -27,7 +27,12 @@ export const EnvReplacerPlugin = (): Plugin => {
       if (s) {
         return {
           code: s.toString(),
-          map: s.generateMap({ hires: true, source: id }),
+          map: s.generateMap({
+            hires: true,
+
+            // Remove possible query parameters, e.g. vue's "?vue&type=script&src=true&lang.ts"
+            source: id.split('?')[0],
```

Comment:
We have `cleanUrl` utility now in `vite-node/utils`

Reply:
Now using `cleanUrl`. Maybe the coverage provider packages should be refactored to use it also. As that's out of scope here, I'll leave it to other day.